src/renderer/App.tsx

```diff
@@ -37,29 +7,37 @@ import {
   WalletModalProvider,
   WalletMultiButton,
 } from '@solana/wallet-adapter-react-ui';
-import { ElectronAppStorageWalletAdapter } from './wallet-adapter/electronAppStorage';
-
-import Account from './nav/Account';
-import Anchor from './nav/Anchor';
-import Validator from './nav/Validator';
-import ValidatorNetworkInfo from './nav/ValidatorNetworkInfo';
-
-import { useAppDispatch, useAppSelector } from './hooks';
+// Default styles that can be overridden by your app
+import '@solana/wallet-adapter-react-ui/styles.css';
+import * as sol from '@solana/web3.js';
+import isElectron from 'is-electron';
+import { FC, useMemo, useState } from 'react';
+import { Button, Form } from 'react-bootstrap';
+import Container from 'react-bootstrap/Container';
+import OverlayTrigger from 'react-bootstrap/OverlayTrigger';
+import Tooltip from 'react-bootstrap/Tooltip';
+import { NavLink, Outlet, Route, Routes } from 'react-router-dom';
+import { ToastContainer } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import './App.scss';
+import { getElectronStorageWallet } from './data/accounts/account';
+import { useAccountsState } from './data/accounts/accountState';
 import {
-  useConfigState,
-  setConfigValue,
   ConfigKey,
```

Comment:
I presume these ordering changes are because your local prettifier settings are arguing with ours.

I'd love these settings to be in the project only - so there's no chance that a user's personal choices bleed over.

Reply:
Argh, yeah this is always a classic frustrating bit with frontend dev - there's so many tools that could interfere with the formatting guidelines. A pre-commit hook would probably be the best approach since it enforces that everything is correct right at the end. I've noticed that there seems to be one but it doesn't run for some reason. I'll look into it.